coordinator/src/main/scala/filodb/coordinator/v2/FiloDbClusterDiscovery.scala

```diff
@@ -122,13 +141,20 @@ class FiloDbClusterDiscovery(settings: FilodbSettings,
   private val datasetToMapper = new ConcurrentHashMap[DatasetRef, ShardMapper]()
   def registerDatasetForDiscovery(dataset: DatasetRef, numShards: Int): Unit = {
     require(failureDetectionInterval.toMillis > 5000, "failure detection interval should be > 5s")
-    logger.info(s"Starting discovery pipeline for $dataset")
+    logger.info(s"[ClusterV2] Starting discovery pipeline for $dataset")
```

Comment:
Same as above: these will log even during V1 clustering?

Reply:
Nope, this is exclusive to V2